parm/gdas/snow_finalize_ens_update.yaml.j2

```
@@ -0,0 +1,70 @@
+######################################
+# set some variables
+######################################
+{% set PDY = current_cycle | to_YMD %}
+{% set cyc = current_cycle | strftime("%H") %}
+copy:
+######################################
+# copy analyses to directories
+######################################
+{% for mem in range(1, NMEM_ENS + 1) %}
+# define variables
+# Declare a dict of search and replace terms to run on each template
+{% set tmpl_dict = {'ROTDIR':ROTDIR,
+                    'RUN':RUN,
+                    'YMD':PDY,
+                    'HH':cyc,
+                    'MEMDIR':"mem" + '%03d' % mem} %}
+
+# Replace template variables with tmpl_dict, one key at a time
+# This must be done in a namespace to overcome jinja scoping
+#  Variables set inside of a for loop are lost at the end of the loop
+#  unless they are part of a namespace
+{% set com_ns = namespace(COM_SNOW_ANALYSIS_MEM = COM_SNOW_ANALYSIS_TMPL) %}
+
+{% for key in tmpl_dict.keys() %}
+{% set search_term = '${' + key + '}' %}
+{% set replace_term = tmpl_dict[key] %}
+{% set com_ns.COM_SNOW_ANALYSIS_MEM = com_ns.COM_SNOW_ANALYSIS_MEM.replace(search_term, replace_term) %}
+{% endfor %}
+{% for tile in range(1, 7) %}
+- ["{{ DATA }}/anl/mem{{ '%03d' % mem }}/{{ current_cycle | to_fv3time }}.sfc_data.tile{{ tile }}.nc", "{{ com_ns.COM_SNOW_ANALYSIS_MEM }}/{{ current_cycle | to_fv3time }}.sfc_data.tile{{ tile }}.nc"]
+{% endfor %}
+{% if DOIAU %}
```

Suggest checking for `True`.  If `DOIAU` is undefined, it will be set as `Undefined`, which will evaluate as `True`.
```suggestion
{% if DOIAU == True %}
```

---

Thanks for this, I didn't know this about Jinja